### PR TITLE
Pass additional args via reply.nextRender() method

### PR DESCRIPTION
**Pull request by vijayrajasekaran**
Hello,

While `app.render(req.raw, reply.raw, path, queryParams)` method is working good but I am unable to send additional params via `reply.nextRender(path, queryParams)` method. Let me know if there is any other way or if this fix is good enough.

Thanks.

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -97,7 +97,12 @@ function fastifyNext (fastify, options, next) {
       reply.raw.setHeader(headerName, headerValue)
     }
 
-    await app.render(request.raw, reply.raw, path, request.query)
+    if (typeof args === undefined) {
+      await app.render(request.raw, reply.raw, path, request.query)
+    } else {
+      delete request.query
```
Comment (mcollina) [inline index.js:103]: this should not be deleted

**Review by mcollina — COMMENTED** [review]
Can you please add a unit test?

File: index.js (hunk 1 above)
Comment (vijayrajasekaran) [reply index.js:103]: I will remove the delete operation. I tried the below test for evaluating this commit, but it fails. The additional params are not available in the `res` object itself.

```
test('should decorate with next render function passing additional args/query params', async t => {
  // t.plan(2)

  const fastify = Fastify()
  t.teardown(() => fastify.close())

  await fastify.register(require('./index'))

  fastify.addHook('onRequest', (req, reply, done) => {
    reply.header('test-header', 'hello')
    done()
  })

  fastify.get('/hello', (req, reply) => {
    return reply.nextRender('/hello', { msg: 'hi' })
  })

  const res = await fastify.inject({
    url: '/hello',
    method: 'GET'
  })

  t.equal(res.statusCode, 200)
  t.equal(res.headers['test-header'], 'hello')
  t.equal(res.raw.req.url, '/hello?msg=hi')
})
```

Comment (mcollina) [reply index.js:103]: Look in https://github.com/fastify/light-my-request for which properties are available.

**Pull request closed without merging**